Hotels Link added below FAQ

dhuadaar opened the pull request:
Link would take people directly to OYO and Trivago with availability on
conference dates and distance filter of 5 KM

dhuadaar committed:
Hotels Link added below FAQ

Link would take people directly to OYO and Trivago with availability on
conference dates and distance filter of 5 KM

satyaakam commented:
@PeeyushAgg make it in the Q&A format like where can we stay while for the conference ? then answer with the links same for travel as well how do i travel from Airport,Railway station etc...

dhuadaar committed:
Accomodation and Travel

Separate FAQ for travelling and accomodation created. Accomodation has
links to OYO and Trivago with fitters of 5km near JNU and 5-6 March
check in check out.

Travelling ques has nearby metro stations listed along with a GPS
location of convention center for people taking cab.

dhuadaar commented:
@satyaakam 

Edits made. PR sent.
